storage/app/build.gradle.kts

```
@@ -47,7 +47,7 @@ dependencies {
     // When using the BoM, you don't specify versions in Firebase library dependencies
     implementation("com.google.firebase:firebase-storage")
 
-    implementation("com.firebaseui:firebase-ui-storage:8.0.2")
+    implementation("com.firebaseui:firebase-ui-storage:9.0.0")
```

This is hopefully the last time I need to do this

```suggestion
    implementation("com.firebaseui:firebase-ui-storage:8.0.2")
```